packages/docs/pages/input.tsx

```diff
@@ -1,98 +1,147 @@
 import { Form, FormGroup, H1, Input, Panel, Text } from '@bigcommerce/big-design';
 import { CheckCircleIcon } from '@bigcommerce/big-design-icons';
-import React, { useState } from 'react';
+import React, { Fragment, useState } from 'react';
 
-import { Code, CodePreview, PageNavigation } from '../components';
+import { Code, CodePreview, ContentRoutingTabs, GuidelinesTable, List, NextLink } from '../components';
 import { InputPropTable } from '../PropTables';
 
 const InputPage = () => {
-  const items = [
-    {
-      id: 'examples',
-      title: 'Examples',
-      render: () => (
-        <>
-          <Panel>
-            <Text>Inputs are stylized form controls with the ability of controling validation.</Text>
-            <CodePreview>
-              {/* jsx-to-string:start */}
-              {function Example() {
-                const [value, setValue] = useState('');
+  return (
+    <>
+      <H1>Input</H1>
 
-                const handleChange = (event) => setValue(event.target.value);
+      <Panel header="Overview" headerId="overview">
+        <Text>
+          <Code primary>Inputs</Code> are form controls that allow the entry of numeric or text data, often supported by
+          system validation.
+        </Text>
+        <Text bold>When to use:</Text>
+        <List>
+          <List.Item>Use text inputs to let the user input short lines of text or numbers.</List.Item>
+          <List.Item>
+            Typically used within <NextLink href="/form">Forms</NextLink>.
```

Comment:
Umm... I would argue that they should _always_ be used in forms.

Reply:
I think its valid HTML5 to put them outside ?

Reply:
It's valid, but when would not put one inside a form?

Reply:
Tried to fiend a official resource that explicity says that is ok to use it without a form, but I didn't. However, I saw some examples of using the <input/> outside a form. 
Some references:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/search
https://html.spec.whatwg.org/multipage/input.html

I think would be more semantic correct if we use the input inside a form.

Reply:
> It's valid, but when would not put one inside a form?

The top channel-select for cp apps that are multi channel comes to mind. Essentially, I think design is exploring a page level header that provides the ability to quickly swap between channels. Its kind of a top level control that essentially re-renders the entire app with different data, and arguably is not technically a form

Reply:
I'd say that should be inside a form element still. It tells something to go fetch new data or sorts.

Also, encapsulating form controls inside forms provides encapsulation, especially if there are two sets of form controls on a page (channel swapping vs. the form and data it controls/renders)

Reply:
> I'd say that should be inside a form element still. It tells something to go fetch new data of sorts.

Hmm, lots of other things fetch data and are not inside forms. I'm not sure that just this by itself justifies the need for a form

> Also, wrapping form controls inside forms provides encapsulation, especially if there are two sets of form controls on a page (channel swapping vs. the form and data it controls/renders)

As long as all the form controls on the page are correctly ID-ed and labeled, and a11y accounted for, I would assume that's sufficient encapsulation

To be clear, I'm all in for forms. Kinda just playing devil's advocate here because the term _should always be used inside forms_ is quite harsh IMO

Reply:
>To be clear, I'm all in for forms. Kinda just playing devil's advocate here because the term should always be used inside forms is quite harsh IMO

Totally 😂 Probably shouldn't have used _always_, but "Typically" seems too light-hearted. Maybe it should be _most of the time_ 🤷‍♂️ 

There are definitely a few unique cases which probably you don't need a form, but in most cases you should use a form - you get a lot more benefit using them then not.

Reply:
or could it be `Frequently use...` ?

Reply:
Yeah I would probably go with `Most of the time...`

Reply:
Note: I updated `line 22` with the following suggestion: `Most of the time...`